templates/testpmd/README.md

```diff
@@ -95,9 +95,11 @@ Note that `containerDisk` value should be replaced with the actual repo where th
 
 
 ## Extra steps on the running VM
-DPDK requires that the driver bound to the interface that uses it (the SR-IOV interface in the VM) is vfio-pci.
+DPDK requires that the driver bound to the interface that uses it (the SR-IOV interface in the VM) is one of few models.
+If this driver model is iavf, then the VF interface (i.e. the SR-IOV NIC in the VM) should be actively bound to an vfio-pci driver by the user.
```

Comment:
I would omit "model" it may confuse the reader since this sentence refers to the VF driver, and rephrase to something like:
If the VF interface driver is `iavf` it should be changed to `vfio-pci` manually.